app/src/main/kotlin/com/wire/android/ui/home/conversations/model/UIMessage.kt

```diff
@@ -212,6 +212,12 @@ sealed class UIMessageContent {
         override val deliveryStatus: DeliveryStatusContent = DeliveryStatusContent.CompleteDelivery
     ) : Regular(), PartialDeliverable
 
+    data class Composite(
+        val messageBody: MessageBody?,
+        val buttonList: List<MessageButton>,
+        override val deliveryStatus: DeliveryStatusContent = DeliveryStatusContent.CompleteDelivery
+    ) : Regular(), PartialDeliverable
```

Comment:
```suggestion
    ) : Regular()
```

No need to implement `PartialDeliverable` as this type is out of scope for partial deliverable.

Reply:
my assumption is since composite messages contain text it should be handled as text
anyway there is no way in the app to send composite messages 👍🏼